farm/infer.py

```diff
@@ -293,16 +294,16 @@ def inference_from_file(self, file, multiprocessing_chunksize=None, streaming=Fa
             return list(preds_all)
 
     def inference_from_dicts(
-        self, dicts, rest_api_schema=False, multiprocessing_chunksize=None, streaming=False
+        self, dicts, return_json=True, num_processes=None, multiprocessing_chunksize=None, streaming=False
     ):
         """
         Runs down-stream inference on samples created from input dictionaries.
         The format of the input `dicts` depends on the task:
 
-        * QA: [{"qas": ["What is X?"], "context":  "Some context containing the answer"}]
+        * QA (internal):    [{"qas": ["What is X?"], "context":  "Some context containing the answer"}]
```

Comment:
These two formats are so similar, I actually think it would be best if we just had the one input format. I think we should keep the Squad style dicts but call them something that is not specific to Squad